### ISSUE TITLE
Positional Attributes are not honoured with Asciidoctor 2.0.0

### ISSUE THREAD

**Issue by robertpanzer** [asciidoctor/asciidoctor-diagram#219]
To reproduce install asciidoctor 2.0.0 and asciidoctor-diagram 1.5.15.
Then create the following document:
```adoc
= Test

Test

== Test

[plantuml,test,svg]
----
A --> B
----

```

Render the document with `asciidoctor -r asciidoctor-diagram test.adoc`.
Then check the resulting html.
It contains a reference to a png with a random name instead of `test.svg`:
```html
<img src="diag-7f08224f605e107120b662e8be428889.png" alt="diag 7f08224f605e107120b662e8be428889" width="78" height="111">
```

I guess the reason for this is that the option for positional attributes was renamed from `:pos_attrs` to `:positional_attrs` and that it has to be fixed here: 
https://github.com/asciidoctor/asciidoctor-diagram/blob/bcc4b061f37222e7ef76823feab8893ec1d708a0/lib/asciidoctor-diagram/extensions.rb#L297
and
https://github.com/asciidoctor/asciidoctor-diagram/blob/bcc4b061f37222e7ef76823feab8893ec1d708a0/lib/asciidoctor-diagram/extensions.rb#L315

**Comment by pepijnve** [comment]
This is caused by a very last minute change in asciidoctor 2.0.0 indeed. I'll get a compatibility fix in place asap.

**Comment by pepijnve** [comment]
Fixed in 1.5.16

**Comment by mojavelinux** [comment]
Argh, I didn't mean to introduce this breaking change. I checked the builds, but I got duped by the queue of builds. Travis was reporting the build as green, but the build that would fail was still pending.

Now I realize that existing code could be depending on this option name, so I'm going to put a patch in 2.0.1. I don't want to force existing extensions to change.

**Comment by mojavelinux** [comment]
There's a far simpler way to solve this problem. Just register both options.

```
subclass.option :pos_attrs, ['target', 'format']
subclass.option :positional_attrs, ['target', 'format']
```

**Comment by pepijnve** [comment]
@mojavelinux no worries; we spotted it quickly enough. I went for a really quick fix. I’ll come up with something more elegant for diagram v2 (basically the server branch I’m working on).

**Comment by mojavelinux** [comment]
My plan is for you to be able to remove this fix once 2.0.1 is out. Users will have to use 2.0.1 instead of 2.0.0, but I think that's a reasonable expectation.
